phoenixRest/models/crew/application.py

```diff
@@ -73,7 +76,7 @@ def __json__(self, request):
             'uuid': str(self.uuid),
             'crews': self.crews,
             'event': self.event,
-            'user': map_user_with_secret_fields(self.user, request),
```

Comment:
why did you change this

Reply:
In general I don't want `__json__` to return a lot of data. This is pretty expensive, and generates many queries.